src/test/kotlin/jces1209/vu/TakeScreenshotHookTest.kt

```diff
@@ -0,0 +1,55 @@
+package jces1209.vu
+
+import com.atlassian.performance.tools.jiraactions.api.ActionMetric
+import com.atlassian.performance.tools.jiraactions.api.ActionResult
+import org.assertj.core.api.Assertions
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import org.openqa.selenium.OutputType
+import org.openqa.selenium.TakesScreenshot
+import java.time.Duration
+import java.time.Instant
+
+
+class TakeScreenshotHookTest {
```

Comment:
Is it intentionally not in `custom-vu/src/test`?

Reply:
Intellij generated the test in the main project, and I didn't spot it. I'll move it.